server/src/controllers/group.controller.ts

```diff
@@ -142,4 +143,21 @@ const joinGroup = async (req: Request, res: Response, next: NextFunction) => {
   }
 };
 
-export default { createGroup, getGroupMembers, createChannel, joinGroup };
+const deleteGroup = async (req: Request, res: Response, next: NextFunction) => {
+  const { id } = req.params;
+  const userID = req.session.userID;
```

Comment:
```suggestion
  const { userID } = req.session;
```
다른 곳에서 이런 식으로 많이 받아가지고!! 추천해봅니다